app/components/timeline-point.js

```diff
@@ -105,7 +105,6 @@ const useStyles = createUseStyles(theme => ({
     description: {
         color: theme.colorSecondary,
         fontSize: theme.secondaryTextFontSize,
-        opacity: theme.secondaryTextOpacity,
```

Comment:
It's awesome that there is a one line fix for this. Thanks for figuring it out. 

One of the things I've learned is the best comments are about **why** a piece of code is there - (when it's not obvious).  I think you should add a comment here that you added this because the react-calendar background was wacky without it.  (or something like that).

But, I saw that when you pick the date on the bottom most timeline entry, with the footer is below that (using the the dev server), then the whole background of the datepicker turns purple and is hard to read. 

I don't know why someone thought it would be a good idea to make the background color dependent on what it's on top of, but lets take that out.

 in election-date-input for the datePicker style, there is
```
        '@supports (backdrop-filter: blur(6rem))': {
            backdropFilter: 'blur(6rem)',
            background: '#fcfdff00',
        },
```
lets take that out.

Then, if we take that out, do we still need to set the opacity of the description?  If so, add the comment, if not take out the line.

Thanks.

Reply:
I actually **removed** the opacity line from the description text. As I was troubleshooting, I realized the description text was the only element that was appearing on top of the calendar and upon removing the opacity CSS property from the class the issue was fixed. I can't seem to find why that was there in the first place and did not see any adverse effects after removing.

I too initially thought the blur effect was the culprit, but removing it did not solve the opacity issue. I do agree it is a good idea to remove. It can have a cool looking effect, but it really depends on the background. It's also not fully supported by all browsers.

So this latest commit removes the blur effect, but I did not add any comments to the background text CSS since there is no longer something to comment about. Although, if I misunderstood anything, please let me know.